server/src/CompilationDatabase.cpp

```diff
@@ -0,0 +1,75 @@
+/**
+ * \file
+ *
+ * \brief Using libclang for reading the JSON compilation database.
+ *
+ * This file is distributed under the GNU General Public License. See
+ * COPYING for details.
+ */
+#include <iostream>
```

Comment:
Order of includes is reversed here, I go from the most specific to the less specific.

See http://llvm.org/docs/CodingStandards.html#include-style